demos/common/samples/args_helper.hpp

```diff
@@ -77,7 +88,7 @@ inline void parseInputFilesArguments(std::vector<std::string> &files) {
     std::vector<std::string> args = gflags::GetArgvs();
     bool readArguments = false;
     for (size_t i = 0; i < args.size(); i++) {
-        if (args.at(i) == "-i" || args.at(i) == "--images") {
+        if (args.at(i) == "-i") {
```

Comment:
What's the reason for this change?

Reply:
Demos that use this function will accept both `-i` and `--images` instead of just `-i`.

Reply:
OK, but then you also need to update the comment above it.

Reply:
I would completely remove that comment.

Reply:
Updated